graylog2-web-interface/src/views/components/Query.jsx

```diff
@@ -56,7 +58,27 @@ const _renderWidgetGrid = (widgetDefs, widgetMapping, results, positions, queryI
   );
 };
 
+const _emptyDashboardInfo = () => {
+  return (
+    <Row className="content" style={{ marginRight: 0, marginLeft: 0 }}>
+      <Col md={12}>
+        <Jumbotron style={{ marginBottom: 0 }}>
+          <h2>This dashboard has no widgets yet</h2>
+          <p>
+            You can create a new widget by selecting a widget type in the left sidebar section &quot;Create&quot;.<br />
+            Have a look at the <DocumentationLink page={DocsHelper.PAGES.DASHBOARDS} text="documentation" />, to learn more about the widget creation.
+          </p>
+        </Jumbotron>
```

Comment:
I used the Jumbotron because I wanted to have the same font size, like the message when creating a new aggregation. The Jumbotron is responsible for the font-size. I have the feeling the Jumbotrone is the wrong component for especially the first case. Whats the preferred way to deal with different font-sizes. Shall I evaluate a possible solution, because I think we will want to use this font-size for many more cases, but without the Jumbotrons margin and padding.

Reply:
Answer: Yes in the mentioned cases we use the Jumbotron mainly for the font size, I will talk with @kyleknighted about a general font-size concept